### ISSUE TITLE
refactor: obtain output from TSC repo linter

### ISSUE THREAD

**Issue by petermetz** [hyperledger/cactus#662]
### Description

We need to get going with adopting the TSC repo structure recommendations where applicable and/or send PRs to the default config file (rule set parameter) where that's more suiting. 

### Acceptance Criteria
1. Run the tool against the `main` branch of the repository
2. Post the output here, whatever it may be
3. Initiated discussion on which errors/warnings can we fix and which ones we'd like to squash by updating the repo linter rule set instead.
4. Created follow-up tasks for addressing each item as discussed during `3)`

### For Context the original email

> Hi all,
> 
> The TSC decided to recommend that all Hyperledger projects adopt a common repository structure to provide some consistency across projects and make it easier for people to find their way around and quickly locate important information. In order to help implement it the TSC then decided to leverage the TODO group repolinter tool.
> 
> A customized configuration file has been developed for Hyperledger and I'd like to invite all the projects to try and use it.
> 
> Contrary to what was first thought, we do not want every project to copy that file into their own repos. This is bound to create a maintenance nightmare. So, instead, we'd like to try and use the same common config file.
> 
> After installing repolinter you can check your repository with the following command from the root directory of your repo:
> 
> repolinter --rulesetUrl https://github.com/hyperledger-labs/hyperledger-community-management-tools/raw/main/repo_structure/repolint.json
> 
> It is understood that the config file may not work well for all projects. We welcome contribution on how to improve it. All PRs welcome. :-) Just keep in mind that the goal is to make the file work for everyone so don't simply remove stuff that's not relevant to your repo. Some errors can simply be ignored.
> 
> Finally, whether this should be integrated in your CI is up to you. As a first step it may be enough to give it a run every now and then.
> 
> Please, let us know how your experiment goes. We're interested in both good and bad stories!

cc: @takeutak @sfuji822 @hartm @jonathan-m-hamilton @AzaharaC @jordigiam @kikoncuo @lehors

**Comment by Leeyoungone** [comment]
May I take this one?

**Comment by petermetz** [comment]
> May I take this one?

@Leeyoungone It's yours!

**Comment by Leeyoungone** [comment]
Here's the [output](https://github.com/ryjones/rl-report/blob/main/main/cactus-repolinter-report.md#repolinter-report) (previously done) after running the tool against the main branch of the repository. 

We currently have: 
_Fail_
❌ security-file-matches: SECURITY.md (file does not exist) 
_Warning_
⚠️ notice-file-exists: NOTICE* (file does not exist) 
⚠️ source-license-headers-exist:14 file(s) not containing the pattern(s): Copyright, License.
⚠️ package-metadata-exists: go.mod (file does not exist in main)

We could either add a SECURITY.md file to get rid of the error or squash by updating the repo linter rule. As for the warnings that exist because we don't have these files, the same concept or whether to add the files or update the repo linter rule applies. 

On a side note, If we do go about creating files to address these repolinter outputs, I'm thinking that maybe I could make issue (or issues) to address each one. Please let me know what you think @petermetz !

**Comment by hartm** [comment]
We should have a security reporting policy in place, and that's what the security.md file is supposed to address.  We may need to discuss how this should be handled before we add a security.md file.

**Comment by petermetz** [comment]
Agreed. As a step 0 I need to figure out why I'm still not getting the emails from the security mailing list.

**Comment by petermetz** [comment]
@Leeyoungone Sorry for the slow turnaround on this one. Please create a `SECURITY.md` file in the project root with the contents being equivalent with what's here: https://raw.githubusercontent.com/hyperledger/fabric/main/SECURITY.md

No need to create a follow-up task because this is the only item necessary to make #662 closable so if you send a PR with the above then that should do it for this issue and I'll work in parallel to get access to the security mailing list.
